src/tests/EdFi.DataManagementService.Tests.E2E/Features/References/UpdateInvalidReferenceValidation.feature

```diff
@@ -0,0 +1,226 @@
+Feature: Invalid Reference Validation
```

Comment:
Rename feature to `Update Reference Validation` and file to `UpdateReferenceValidation.feature`